From bb982eea487478ba7dc42e1dc43c0c321feeba99 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 11 Jul 2021 18:24:04 -0400
Subject: [PATCH 16/18] Revert "SERVER-45772 Require VS 2019 to build the
 server"

This reverts commit 7160266ea5d1cf497570ba77ba148072ab1f6bd3.
---
 SConstruct | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SConstruct b/SConstruct
index 20324fa7b4..3272d5ad64 100644
--- a/SConstruct
+++ b/SConstruct
@@ -2610,14 +2610,14 @@ def doConfigure(myenv):
     # bare compilers, and we should re-check at the very end that TryCompile and TryLink still
     # work with the flags we have selected.
     if myenv.ToolchainIs('msvc'):
-        compiler_minimum_string = "Microsoft Visual Studio 2019 16.4"
+        compiler_minimum_string = "Microsoft Visual Studio 2017 15.9"
         compiler_test_body = textwrap.dedent(
         """
         #if !defined(_MSC_VER)
         #error
         #endif
 
-        #if _MSC_VER < 1924
+        #if _MSC_VER < 1916
         #error %s or newer is required to build MongoDB
         #endif
 
-- 
2.33.1

